cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerClusterStatusService.java

```diff
@@ -440,7 +445,7 @@ public boolean isClusterManagerRunning() {
     @Override
     public boolean isClusterManagerRunningQuickCheck() {
```

Comment:
You don't need a 'quickcheck' method anymore.

Reply:
Yes that's right, I will remove it.